R/lme.R

```diff
@@ -179,16 +179,16 @@ lme_coefs <- function(x, y, ...) {
   if (length(dots) > 0) {
     args <- c(args, dots[[1]])
   }
-
-  if (!is.factor(y[[1]])) {
-    cl <- rlang::call2("lmer", .ns = "lme4", !!!args)
-    mod <- rlang::eval_tidy(cl)
-  } else {
+  
+  if (is.factor(y[[1]])) {
     args$data$y <- as.numeric(args$data$y) - 1
-    args$family <- stats::binomial
-    cl <- rlang::call2("glmer", .ns = "lme4", !!!args)
-    mod <- rlang::eval_tidy(cl)
+    if (is.null(args$family)) {
```

Comment:
Are there tests for `step_lencode_mixed()` with Poisson?

Reply:
yes https://github.com/tidymodels/embed/pull/127/files#diff-0c5f55359eb319fd39087413214a46659d60be6fa28cfa62066fe7621517f042